node-api/src/metadata.rs

```diff
@@ -1,31 +1,22 @@
-// Copyright 2019-2021 Parity Technologies (UK) Ltd. and Supercomputing Systems AG
-// and Integritee AG.
```

Comment:
Recopy paste from subxt and simplified / made `no_std` compatible where necessary